src/system_descriptor_data.jl

```diff
@@ -1411,7 +1411,6 @@ const SYSTEM_CATALOG = [
         supported_arguments = [
             SystemArgument(;
                 name = :avr_type,
-                default = "AC1A",
```

Comment:
@jd-lara Can you confirm that none of these descriptors should have default values?

Reply:
In my opinion, those descriptors should not have default values. But I did not make that descriptor data, so I'm not sure if there was a reason of having default values there.

Reply:
The main branch does not have defaults.

Reply:
Agree with Rodrigo